docs/synapse/userguides/storm_ref_automation.rstorm

```diff
@@ -157,131 +349,211 @@ Triggers automatically have the Storm variable ``$auto`` populated when they run
 
+Because cron jobs are scheduled, they are most appropriate for automating routine, non-urgent tasks; maintenance
+tasks; or resource-intensive tasks that should run during off-hours.
 
-.. storm-cli:: trigger.add --name "enrich osint fqdn" tag:add --form inet:fqdn --tag osint --query { | whois | pdns | malware }
+- **Data ingest.** Cron jobs can be used to ingest / synchronize data that you want to load into Synapse on a
+  regular basis. For example, you can create a cron job to retrieve and load a list of TOR exit nodes every hour.
 
+- **Housekeeping.** You created a trigger to automatically look up and apply geolocation and autonomous system
+  (AS) properties to IPv4 nodes when they are created in Synapse. However, you already have a large number of
+  IPv4 nodes that existed before the trigger was added. You can create a one-time cron job to retrieve and
+  "backfill" this information for IPv4s that already exist.
 
-- Add a trigger that fires when the tag ``#osint`` (indicating that the node was listed as a malicious indicator in public reporting) is applied to **any** node. The trigger should call (execute) a macro called ``enrich``. The macro contains a Storm query that uses a switch case to call the appropriate extended Storm commands based on the tagged node’s form (e.g., perform different enrichment / call different third-party services based on whether the node is an FQDN, an IPv4, an email address, a URL, etc.).
+- **Process intensive jobs.** Data enrichment may be resource intensive where it generates a significant number
+  of write operations. If you reguarly perform routine (non-urgent) enrichment, it can be scheduled to run
+  when it will have less impact on users.
+  
 
+Usage Notes
+~~~~~~~~~~~
 
-.. storm-pre:: $pkg=$lib.dict(name='docs', version='0.0.1', commands=($lib.dict(name=whois, storm=${} ),)) $lib.print($pkg) $lib.pkg.add($pkg)
-.. storm-pre:: $pkg=$lib.dict(name='docs', version='0.0.1', commands=($lib.dict(name=malware, storm=${} ),)) $lib.print($pkg) $lib.pkg.add($pkg)
-.. storm-pre:: $pkg=$lib.dict(name='docs', version='0.0.1', commands=($lib.dict(name=revwhois, storm=${} ),)) $lib.print($pkg) $lib.pkg.add($pkg)
-.. storm-pre:: macro.set enrich ${ switch $node.form() {/* You can put comments in macros!!! */ "inet:fqdn": { | whois | pdns | malware } "inet:ipv4": { | pdns } "inet:email": { | revwhois } *: { } } }
-.. storm-cli:: trigger.add --name "enrich osint" tag:add --tag osint --query { | macro.exec enrich }
+- Users must be granted permissions in order to be able to work with cron jobs (i.e., to execute the
+  associated ``cron.*`` Storm commands).
 
+- Cron jobs execute **with the permissions of the user who created the job**. If a cron jobs calls a
+  macro, the macro will execute with the permissions of the cron job (macros execute with the
+  permissions of the calling user).
 
-.. _auto-cron:
+- Cron jobs are **view-specific** - they reside (and execute within) a particular :ref:`gloss-view`.
```

Comment:
cron jobs do execute in a particular view, but they actually reside in the cortex. (not sure if that difference is worth including here, but that's why they don't automatically get deleted like triggers do when the view is merged/deleted)

Reply:
Bwaaaah so many things make sense now :smile: